frontend/components/utils/DataProvider.js

```diff
@@ -63,70 +63,66 @@ const getTransactionsInfo = async () => {
   }
 };
 
-const DataProvider = props => {
-  const networks = ["Testnet One", "Testnet Two"];
+const processTransactions = transactions => {
```

Comment:
Why do we need this separated from the components? I suggest we implement this logic in the relevant `transaction/*.js` components.

Reply:
Because I was moving all these api calls to their respective api components. Have already moved them in the latest commit.